### PR TITLE
Feature/code cleanups

### LINKED ISSUE #47: PR #44 issues
* navigateTo("/") on unauthorized in every try-catch block
* get rid of unnecessary empty lines
* validate js libararies
* use `});` instead of two lines
* get rid of unused services

### PR DESCRIPTION
**Pull request by meanin**
resolve #47

### CODE REVIEW

File: 1 Presentation layer/CandidateEvaluator.Client/Pages/Category/CreateCategory.cshtml [hunk 1]
```diff
@@ -13,15 +13,22 @@
...
-{
-    var categoryId = await Http.AuthorizedPostJsonAsync<Guid>("api/Category", _categoryModel);
-    UriHelper.NavigateTo($"/category/{categoryId}");
-}
+    private async Task Save()
+    {
+        try
+        {
+            var categoryId = await Http.AuthorizedPostJsonAsync<Guid>("api/Category", _categoryModel);
+            UriHelper.NavigateTo($"/category/{categoryId}");
```
Comment (plaumen) [inline 1 Presentation layer/CandidateEvaluator.Client/Pages/Category/CreateCategory.cshtml]: Navigation to specific sites repeats. I think method extraction could be useful,

File: 1 Presentation layer/CandidateEvaluator.Server/Startup.cs [hunk 2]
```diff
@@ -92,14 +92,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ICommandHandler<CreateInterviewResult>, CreateInterviewResultHandler>();
             services.AddTransient<ICommandHandler<DeleteInterviewResult>, DeleteInterviewResultHandler>();
 
-            services.AddTransient<IQueryHandler<GetAllCategories, List<Category>>, GetAllCategoriesHandler>();
+            services.AddTransient<IQueryHandler<GetAllCategories, IEnumerable<Category>>, GetAllCategoriesHandler>();
```
Comment (plaumen) [inline 1 Presentation layer/CandidateEvaluator.Server/Startup.cs]: why IEnumerable? I think ICollection will be more useful than IEnumerable

File: 1 Presentation layer/CandidateEvaluator.Client/Pages/Category/CreateCategory.cshtml (hunk 1 above)
Comment (meanin) [reply 1 Presentation layer/CandidateEvaluator.Client/Pages/Category/CreateCategory.cshtml]: The rule of three?

File: 1 Presentation layer/CandidateEvaluator.Server/Startup.cs (hunk 2 above)
Comment (meanin) [reply 1 Presentation layer/CandidateEvaluator.Server/Startup.cs]: ICollection is mutable

**Pull request merged**